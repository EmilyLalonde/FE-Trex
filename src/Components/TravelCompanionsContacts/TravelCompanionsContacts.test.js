import React from "react";
import "react-native";
import TravelCompanionsContacts from "./TravelCompanionsContacts";
import { shallow } from "enzyme";

describe("TravelCompanionsContacts", () => {
  it("should return true", () => {
    expect(true).toEqual(true);
  });
  it("should match the snapshot when loading", () => {
    let wrapper = shallow(<TravelCompanionsContacts navigation={{ navigate: jest.fn() }} />);

    expect(wrapper).toMatchSnapshot();
  });
});
