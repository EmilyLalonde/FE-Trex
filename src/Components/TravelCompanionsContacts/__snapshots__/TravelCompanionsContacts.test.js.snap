// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TravelCompanionsContacts should match the snapshot when loading 1`] = `
<View
  style={
    Object {
      "alignItems": "stretch",
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "flex-start",
    }
  }
>
  <Header />
  <ScrollViewMock>
    <Text>
      Display all contacts marked as travel companions for a specific user
    </Text>
  </ScrollViewMock>
  <Footer
    navigate={[MockFunction]}
  />
</View>
`;
